python/destinations/S3/README.md

```diff
@@ -25,6 +25,7 @@ The code sample uses the following environment variables:
 - **s3_folder_per_stream**: Flag to save different streams to different S3 folders.
 - **prefix**: File name prefix.
 - **suffix**: File suffix (e.g. file type extension).
+- **timezone**: A valid timezone e.g. GMT or US/Eastern or Asia/Singapore used for orchestrating time-based batching and file naming.
```

Comment:
Does this follow some international standard such as ISO or IANA? I was interested to see if there is a definitive list for what can be accepted.

Reply:
IANA. Ill add that to the description